### ISSUE TITLE
.stdout usage in task files breaks execurtion

### ISSUE THREAD

**Issue by atothRedHat** [fubarhouse/ansible-role-rust#19]
I've run across issues with the main.yml and setup.yml files where tasks fail because the conditional or attribute value source uses the ".stdout" function call. removing the ".stdout" fixes the issues and the task executes complete as intended.

Expected Outcome: Tasks complete normally
Actual Outcome: Tasks fail

**Comment by fubarhouse** [comment]
@atothRedHat,

Thanks for reporting, I've made some small changes (in branch `issue/19`) to simplify some of the logic and removed a conditional which sets the variable causing the problem - so that it is always set.

If you're able to re-test this on your environment I would be interested in the feedback, and we can merge the changes or revisit as necessary. Please let me know! The changes can be found in branch `issue/19`.

**Comment by atothRedHat** [comment]
nope, issue is still there. If you look at the files, the .stdout call is still made in 2 spots which breaks the run.
setup.yml: 
- name: "Rust | Set $HOME"
  set_fact:
    fubarhouse_user_dir: "{{ shell_home_dir.stdout }}"
  when: fubarhouse_user_dir is not defined

main.yml:
- name: "Install from installer"
  include_tasks: installer.yml
  when:
    - ((build_rust_from_source|bool == false) and
      (rust_version_requirement not in rust_preinstall_version_r.stdout)) or
      (rust_update|bool == true)
